beacon-chain/sync/service.go

```diff
@@ -93,6 +99,8 @@ func (ss *Service) run() {
 	if synced {
 		ss.RegularSync.Start()
 		return
+	} else {
+		ss.failStatus = errors.New("Service is unhealthy")
```

Comment:
Please do not use a failure status for something you can query. 

Your Status method should look something like this:
```go
func (ss *SyncServer) Status() error {
  synced, err := ss.Querier.IsSynced() 
  if err != nil {
    return err
   }
  if !synced {
    return errors.New("not in sync")
  }
}
```